packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -113,6 +148,17 @@ export const CallControls = (props: CallControlsProps & ContainerRectProps): JSX
           {isEnabled(options?.devicesButton) && (
             <Devices displayType={options?.displayType} increaseFlyoutItemSize={props.increaseFlyoutItemSize} />
           )}
+          {
+            /* @conditional-compile-remove(one-to-n-calling) */ /* @conditional-compile-remove(one-to-n-calling) */
+            isEnabled(options?.moreButton) && (
```

Comment:
IIRC we can only hold 5 buttons in the width of the call page on small mobile devices so 6 would be too many:
![image](https://user-images.githubusercontent.com/2684369/184035503-ba634f40-3cd4-4df0-ab45-ecfd9cd04bc9.png)

Can you confirm if on the likes of iphone SE or the galaxy fold narrow screen does this fit?
If not we may need to move people into the overflow (is that breaking change?)

Reply:
<img width="436" alt="Screen Shot 2022-08-10 at 4 33 56 PM" src="https://user-images.githubusercontent.com/9782747/184040734-5f61d339-00e8-4dfe-a637-d4e88bbefeba.png">

Works :)

Reply:
![image](https://user-images.githubusercontent.com/94866715/184233791-3c3291a9-c3f8-4cb4-8024-04cc5223466e.png)
People button is now in more button overflow on mobile since it opens a people pane now and isn't a context menu

Reply:
We should discuss as a team if we want to release this as a 2.0.0 release or just a minor release (we are at the point where we have a long list of breaking changes we want to make so maybe its time)